core/CHANGELOG.T2B1.md

```
@@ -1270,8 +1279,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 [#6104]: https://github.com/trezor/trezor-firmware/pull/6104
 [#6109]: https://github.com/trezor/trezor-firmware/pull/6109
 [#6136]: https://github.com/trezor/trezor-firmware/pull/6136
+[#6138]: https://github.com/trezor/trezor-firmware/pull/6138
```

markdownlint-cli2 (0.22.0)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "#6138"

(MD053, link-image-reference-definitions)